src/borg/archive.py

```diff
@@ -846,7 +846,7 @@ def restore_attrs(self, path, item, symlink=False, fd=None):
             if warning:
                 set_ec(EXIT_WARNING)
             # bsdflags include the immutable flag and need to be set last:
-            if not self.nobsdflags and 'bsdflags' in item:
+            if not self.noflags and 'bsdflags' in item:
```

Comment:
hmm, while we have the less misleading "flags / noflags" on the UI / in the docs, we still have "bsdflags" in the items.

not sure whether we should change this or not, maybe needs some thoughts, esp. about compatibility.

Reply:
Yes, I intentionally left it untouched to not break compatibility.